packages/dashboard/tests/frontend_selenium/tests/test_dedicate.py

```diff
@@ -170,6 +172,55 @@ def test_dedicate_page(browser):
 #     assert dedicate_page.sort_node_price() == sorted(price, reverse=False)
 
 
+def test_filter_resource(browser):
```

Comment:
I ran this test case multiple times and it keeps failing with the following error.

```
tests/test_dedicate.py:193: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
pages/dedicate.py:200: in table_nodes
    self.browser.find_element(By.XPATH, f"{self.table_xpath}[{str(i)}]/td[1]/button").click()
```